### ISSUE TITLE
[Feature Request] Download just one page

### ISSUE THREAD

**Issue by Noriller** [inorichi/tachiyomi#2323]
**App version:** 0.84

**Android version:** 9

**Request:**
Now and then a few pages (usually just one really) is downloaded with error. When that happens I have to delete the whole chapter and either download everything or "stream" the whole thing again.

The request is for the option to download just the one page you need.

**Other details:**

Thanks for everything, love the app. =]

**Comment by MerradOklom** [comment]
¿Have you tried this? Open the chapter from the source you're downloading, go to the page that failed and tap and hold the middle of the screen and do as shown below:
![Photo_20191112_144745](https://user-images.githubusercontent.com/43864503/68696681-30ad1100-055c-11ea-8bd6-b631f131f06d.jpg)

**Comment by Noriller** [comment]
Maybe my wording is wrong, but it's not like that. I'm talking about when the page comes with a error.

**Comment by MerradOklom** [comment]
> Maybe my wording is wrong, but it's not like that. I'm talking about when the page comes with a error.

You said you want to download that one page that failed to download, and that's exactly what I did on my previous comment.

**Comment by Noriller** [comment]
I might be crazy, but doesn't that just save that page on your phone/pictures/tachiyomi?

**Comment by MerradOklom** [comment]
Yeah, after thinking about it this might not be the best solution (because of that and other complications), so forget what I said.

**Comment by Efreak** [comment]
Implementation ideas: explicitly force recheck and download of missing files by:

a. any action that would otherwise cause the app to download the chapter
  - long-pressing and choosing to download a chapter that has already been downloaded
  - by downloading 'unread' (assuming said chapter is, in fact, unread) or 'all' chapters through the menu

b. add 'repair downloads' item to downloads submenu (I think this was done in MangaWatcher)
